### PR TITLE
dev-vcs/easygit: 1.7.5.2 version bump

### PR DESCRIPTION
**Pull request by culot**
Closes: https://bugs.gentoo.org/662254

### CODE REVIEW

File: dev-vcs/easygit/easygit-1.7.5.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="eg"
```
Comment (mgorny) [inline dev-vcs/easygit/easygit-1.7.5.2.ebuild]: Please remove this variable. It doesn't really help with anything, and replaces two characters with eight…

File: dev-vcs/easygit/easygit-1.7.5.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+DESCRIPTION="Easy GIT (eg), a frontend for git designed for former cvs and svn users"
+HOMEPAGE="https://www.gnome.org/~newren/eg/"
+SRC_URI="https://www.gnome.org/~newren/${MY_PN}/download/${PV}/${MY_PN} -> ${PF}"
```
Comment (mgorny) [inline dev-vcs/easygit/easygit-1.7.5.2.ebuild]: PF is incorrect here. Revisions are intended for internal Gentoo use, and must not be used for versioning sources.

File: dev-vcs/easygit/easygit-1.7.5.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~ppc ~x86"
+SLOT="0"
+IUSE=""
+
+RDEPEND=">=dev-vcs/git-${PV}
+	dev-lang/perl"
+
+src_unpack() {
+	mkdir "${S}" || die
```
Comment (mgorny) [inline dev-vcs/easygit/easygit-1.7.5.2.ebuild]: Don't create unnecessary directories, set `S=${WORKDIR}` instead.

File: dev-vcs/easygit/easygit-1.7.5.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+	cp "${DISTDIR}/${PF}" "${S}/${MY_PN}" || die
+}
+
+src_compile() { :; }
```
Comment (mgorny) [inline dev-vcs/easygit/easygit-1.7.5.2.ebuild]: Do you actually need to do this? Unless I'm mistaken, the default would do nothing here.

File: dev-vcs/easygit/easygit-1.7.5.2.ebuild (hunk 2 above)
Comment (monsieurp) [reply dev-vcs/easygit/easygit-1.7.5.2.ebuild]: @culot: use `${P}` instead.

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**